### ISSUE TITLE
In-Game: show_debug_message() showing a try/catch exception struct causes a silent crash

### ISSUE THREAD

**Issue by toby-yoyo** [YoYoGames/GameMaker-Bugs#6190]
### Description

Spotted in a user's full game. The following code will crash in Beta IDE v2024.600.0.569 / Beta Runtime v2024.600.0.596:
```
try {
    show_error("test", false);
}
catch(exception) {
	show_debug_message(exception);
}
```
### Steps To Reproduce

1. Open and run the attached project: [gmb-6190-x.zip](https://github.com/user-attachments/files/15764637/gmb-6190-x.zip)
2. Observe the crash.

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

Beta IDE v2024.600.0.569 / Beta Runtime v2024.600.0.596

### Which platform(s) are you seeing the problem on?

Windows

### Sample Project Added?

- [X] I have included a small sample project

**Comment by rwkay** [comment]
Fixed in 2024.6 (when the above PR is merged)

**Comment by mgeddesGM** [comment]
verified as of  IDE v2024.600.0.577  Runtime v2024.600.0.603

**Comment by YYDan** [comment]
This fix is suspected to cause knock-ons, so has been taken out of 2024.6 now and will return in 2024.8.

**Comment by rwkay** [comment]
Fixed in 2024.8 properly this time...

**Comment by cameron-home** [comment]
Verified in IDE v2024.800.0.588  Runtime v2024.800.0.612.
